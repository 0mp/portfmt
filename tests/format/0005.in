BLA!=		foo
BLA+=		bar
BLA+=		muh
