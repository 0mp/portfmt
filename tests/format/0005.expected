BLA!=		foo
BLA+=		bar muh
